packages/mtn-momo/src/Adaptor.js

```diff
@@ -0,0 +1,98 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * State object
+ * @typedef {Object} HttpState
+ * @property data - the parsed response body
+ * @property response - the response from the HTTP server, including headers, statusCode, body, etc
+ * @property references - an array of all previous data objects used in the Job
+ **/
+
+/**
+ * Options provided to the HTTP request
+ * @typedef {Object} RequestOptions
+ * @public
+ * @property {object} errors - Map of errorCodes -> error messages, ie, `{ 404: 'Resource not found;' }`. Pass `false` to suppress errors for this code.
+ * @property {object} form - Pass a JSON object to be serialised into a multipart HTML form (as FormData) in the body.
+ * @property {object} query - An object of query parameters to be encoded into the URL.
+ * @property {object} headers - An object of headers to append to the request.
+ * @property {string} parseAs - Parse the response body as json, text or stream. By default will use the response headers.
+ * @property {number} timeout - Request timeout in ms. Default: 300 seconds.
+ * @property {object} tls - TLS/SSL authentication options. See https://nodejs.org/api/tls.html#tlscreatesecurecontextoptions
+ */
+
+
+/**
+ * Make a HTTP request to the MTN MoMo API ([view docs](https://momodeveloper.mtn.com/api-documentation))
```

Comment:
I don't think the link text is clear enough. Whose docs are we talking about?

I think a good solution might be this:

Make a HTTP request to the [MTN MoMo API](https://momodeveloper.mtn.com/api-documentation)

Also, I've just followed the link, and where is the actual API reference in their docs? We should link users to the reference of endpoints and query parameters, if it exists.

Maybe the API sandbox is a better place to link to?